.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
```

Comment:
nit: not sure if this might be `.github/`?

Reply:
I configured it the same way like the core repo, let's not add too much PRs here :)